lib/pliny/config_helpers.rb

```diff
@@ -50,12 +50,20 @@ def array(method = nil)
     end
 
     def rack_env
-      if pliny_env == "development" || pliny_env == "test"
+      if app_env == "development" || app_env == "test"
```

Comment:
I wonder if we should respect the value of `app_env` but _also_ `pliny_env` here for a bit longer to give people some opportunity to upgrade?

I'm a little worried that with this if someone bumped their Pliny version behavior based off `rack_env` would change without warning in their environments because they hadn't injected the new `APP_ENV` variable into `.env` and/or their app's config.

Thoughts?